dom.bs

```diff
@@ -3057,9 +3096,9 @@ interface mixin ParentNode {
   readonly attribute Element? lastElementChild;
   readonly attribute unsigned long childElementCount;
 
-  [CEReactions, Unscopable] undefined prepend((Node or DOMString)... nodes);
-  [CEReactions, Unscopable] undefined append((Node or DOMString)... nodes);
-  [CEReactions, Unscopable] undefined replaceChildren((Node or DOMString)... nodes);
+  [CEReactions, Unscopable] undefined prepend((Node or DOMString or TrustedScript)... nodes);
+  [CEReactions, Unscopable] undefined append((Node or DOMString or TrustedScript)... nodes);
+  [CEReactions, Unscopable] undefined replaceChildren((Node or DOMString or TrustedScript)... nodes);
```

Comment:
For the record: these methods should be annotated with a note for web-developers that passed text nodes and strings are internally passed to trusted types' default policy, if needed.